storage/gc_queue.go

```diff
@@ -559,8 +563,8 @@ func RunGC(
 	}
 
 	// Clean up the abort cache.
-	gcKeys = append(gcKeys, processAbortCache(ctx, snap, desc.RangeID, now,
-		abortCacheAgeThreshold, &infoMu, pushTxnFn)...)
+	gcKeys = append(gcKeys, processAbortCache(
+		ctx, snap, desc.RangeID, abortSpanThreshold, &infoMu, pushTxnFn)...)
```

Comment:
Might be nice to have 'GC' in the name: `abortSpanGCThreshold`. It just feel ambiguous without.